docs/examples.md

```diff
@@ -14,9 +29,9 @@ Our team uses it to test new features and run simple scenarios.
 
 ![](https://user-images.githubusercontent.com/220264/56353972-56975080-61db-11e9-8b23-06e8b4620995.png)
 
-End 2 end tests for [Bugira Bugtracker](https://bugira.com) app built with Rails & EmberJS. Bugira is a SaaS application which helps to collect user feedback and transform it into professional bugreports.
+End 2 end tests for [Bugira Bugtracker](https://bugira.com) app built with Rails & EmberJS. Bugira is a SaaS application that helps to collect user feedback and transform it into professional bugreports.
```

Comment:
`helps collecting users' feedbacks and transforming them into professional bug reports.`

Reply:
Could you update the wiki page with examples for that? 
https://github.com/Codeception/CodeceptJS/wiki/Examples
This page was generated from wiki

Reply:
updated that on wiki page.